packages/clerk-js/src/core/resources/Organization.ts

```diff
@@ -146,8 +146,15 @@ export class Organization extends BaseResource implements OrganizationResource {
       }).then(res => new Organization(res?.response as OrganizationJSON));
     }
 
-    const body = new FormData();
-    body.append('file', file);
+    let body;
+    if (typeof file === 'string') {
+      body = JSON.stringify({
+        file,
+      });
```

Comment:
:question: Do we need to stringify here? Just checking.

Reply:
@gkats The reason for this is that we need to specify the type of body data.  
References:
- https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch#supplying_request_options
- https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch#uploading_json_data